frontend/src/pages/modelServing/screens/projects/kServeModal/ServingRuntimeArgsSection.tsx

```diff
@@ -13,48 +13,43 @@ import { UpdateObjectAtPropAndValue } from '~/pages/projects/types';
 import { CreatingInferenceServiceObject } from '~/pages/modelServing/screens/types';
 
 type ServingRuntimeArgsSectionType = {
-  data?: CreatingInferenceServiceObject;
+  data: CreatingInferenceServiceObject;
   setData: UpdateObjectAtPropAndValue<CreatingInferenceServiceObject>;
 };
 
-const ServingRuntimeArgsSection: React.FC<ServingRuntimeArgsSectionType> = () => {
-  const [args, setArgs] = React.useState('');
-
-  return (
-    <FormGroup
-      label="Additional serving runtime arguments"
-      labelIcon={
-        <Popover
-          bodyContent={
-            <div>
-              Serving runtime arguments define how the deployed model behaves. Overwriting
-              predefined arguments only affects this model deployment.
-            </div>
-          }
-        >
-          <Icon aria-label="Additional serving runtime arguments info" role="button">
-            <OutlinedQuestionCircleIcon />
-          </Icon>
-        </Popover>
-      }
-      fieldId="model-server-replicas"
-    >
-      <TextArea
-        placeholder={`--arg\n--arg2=value2\n--arg3 value3`}
-        value={args}
-        onChange={(e) => setArgs(e.target.value)}
-        autoResize
-      />
-      <FormHelperText>
-        <HelperText>
-          <HelperTextItem>
-            {`Enter one argument and its values per line. Overwriting the runtime's predefined
+const ServingRuntimeArgsSection: React.FC<ServingRuntimeArgsSectionType> = ({ data, setData }) => (
+  <FormGroup
+    label="Additional serving runtime arguments"
+    labelIcon={
+      <Popover
+        bodyContent={
+          <div>
+            Serving runtime arguments define how the deployed model behaves. Overwriting predefined
+            arguments only affects this model deployment.
+          </div>
+        }
+      >
+        <Icon aria-label="Additional serving runtime arguments info" role="button">
+          <OutlinedQuestionCircleIcon />
+        </Icon>
+      </Popover>
+    }
+    fieldId="serving-runtime-arguments"
+  >
+    <TextArea
```

Comment:
This TextArea needs either an id or an aria-label - I missed this in the last PR, but it's giving a console warning.

It should probably have the id `"serving-runtime-arguments"` because that's the `fieldId` on the FormGroup.